src/libply/built-in/memory.c

```diff
@@ -141,11 +141,7 @@ static int str_ir_post(const struct func *func, struct node *n,
 	ir_emit_insn(ir, MOV_IMM((int32_t)type_sizeof(n->sym->type)), BPF_REG_2, 0);
 	ir_emit_sym_to_reg(ir, BPF_REG_3, ptr->sym);
 
-#if defined(__riscv) && __riscv_xlen == 64
         ir_emit_insn(ir, CALL(BPF_FUNC_probe_read_kernel_str), 0, 0); 
-#else
-        ir_emit_insn(ir, CALL(BPF_FUNC_probe_read_str), 0, 0); 
-#endif /* defined(__riscv) && __riscv_xlen == 64 */
```

Comment:
I think it's better to squash this commit to the origin change in the second commit.  And please mention it requires 5.5+ kernel.